### PR TITLE
updating readme to make it easier to deploy using docker

### PR DESCRIPTION
**Pull request by keepittechie**
small edit to readme

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -779,8 +779,16 @@ Click the button below, to deploy to Vercel 👇
 
 ### Deploying - Option #3: Docker
 
+Pull Image: `docker pull lissy93/web-check`
```
Comment (Lissy93) [inline .github/README.md:782]: > Pull Image: `docker pull lissy93/web-check`

I don't think this is necessary, as if the image isn't found locally, it'll be pulled automatically during the run command. The exception being, unless they user wants to use a different registry (like GHCR). But in this instance, running that pull command will just download it from DockerHub, same as in the run command.

File: .github/README.md [hunk 2]
```diff
@@ -779,8 +779,16 @@ Click the button below, to deploy to Vercel 👇
...
+
 Run `docker run -p 3000:3000 lissy93/web-check`, then open [`localhost:3000`](http://localhost:3000)
 
+or:
+
+Run `docker -d --name webcheck -p 3000:3000 --restart=always lissy93/web-check`, then open [`localhost:3000`](http://localhost:3000)
```
Comment (Lissy93) [inline .github/README.md:788]: +1 to this. Or maybe a `docker-compose.yml` would be more suitable, as that's what most people are going to use to deploy, and it gets a bit hard to read with many docker flags.

File: .github/README.md (hunk 1 above)
Comment (keepittechie) [reply .github/README.md:782]: Okay cool. You're right!

File: .github/README.md (hunk 2 above)
Comment (keepittechie) [reply .github/README.md:788]: Sorry for the delay. I haven't had time outside of work to address this. Working it now.

**Comment by keepittechie** [comment]
I updated the pull request to include modifications to the docker-compose.yml file and revised the README to provide Docker Compose instructions.

**Review by keepittechie — COMMENTED** [review]
I updated the pull request to include modifications to the docker-compose.yml file and revised the README to provide Docker Compose instructions. Hopefully these changes will work.

**Pull request closed without merging**